Gemfile

```diff
@@ -124,6 +124,7 @@ group :development, :test do
 end
 
 group :development do
+  gem "aasm-diagram", require: false
```

Comment:
I thought I might as well leave it in there in case anyone wants to use in the future, it was quite useful, it's just in the development dependencies so hopefully that's okay?